percona-server/riscv64.patch

```diff
@@ -1,20 +1,36 @@
-diff --git PKGBUILD.orig PKGBUILD
---- PKGBUILD.orig
+--- PKGBUILD
 +++ PKGBUILD
 @@ -9,7 +9,7 @@ _myver=${pkgver/_rel*}
- pkgrel=2
+ pkgrel=1
```

Comment:
Please do not do unnecessary pkgrel changes.

Reply:
@felixonmars This is a necessary pkgrel change, introduced by upstream version bump:

https://github.com/archlinux/svntogit-community/blob/8eecb3f0659414d7f04269e85679148327e6cc98/trunk/PKGBUILD#L9